tailwind.config.js

```diff
@@ -7,10 +7,10 @@ module.exports = {
   ],
   theme: {
     colors: {
-      red: '#E40422',
-      black: '#000000',
-      mittelgruen: '#00AA84',
-      blau: '#324fa3',
+      primary: '#773666',
+      black: '#0A1045',
+      success: '#69AC4C',
+      info: '#5891C3',
```

Comment:
One philosophical note about the naming:

I am more and more thinking that "loading" variable names with meaning such as "primary", "success", and "info" is actually counterintuitive in many cases. If I see a design that has a blue background, my instinct is to apply a color/class that has "blue" in it. Making the detour to "info" seems like more cognitive load where I have to process whether "info" is blue or, say, green.

In some cases the more abstract naming makes sense, I guess, e.g. when building a library that has a customizable color theme.

In our case we had the unfortunate situation that the main color scheme changed late in the project, but I would argue that this is rarely the case.

I am also not suggesting to change the variable names now, but I would be curious about your opinion on this.

Reply:
Great comments. I'm not so sure about this. I totally agree about the fact that naming blue things blue make sense! Here I changed that to make it easier to change the colors in case we are still unhappy with the current designs. @Esshahn expressed some reserves about the font and I thought it would be easier to change the "primary" and "info" color that to rename every variable. But this mentality might be a little short sighted. So like I said, I'm not to sure about this. I guess you're right and we should keep following the tailwind way of naming things.

Reply:
I think your concerns were exactly what happened today with the latest color changes (even though they were minimal). That's why I think it's okay to leave the current variable names as-is. I wouldn't rule out that there might be more adjustments in the near future.

Once things have settled a bit, we could rename the colors to something like `bordeaux`, `green` and `blue`.